src/renderer/react/priceBox/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { getInitialStateRenderer } from 'electron-redux';
+import configureStore from 'common/configure-store';
+import PriceBox from './containers/price-box';
+
+const initialState = getInitialStateRenderer();
+const store = configureStore(initialState, 'renderer');
```

Comment:
Why are we configuring the store in a component file? We should do it when we initialise the store for the first time.